activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -777,6 +777,7 @@ def index_name_exists?(table_name, index_name, default)
       #   The reference column type. Defaults to +:integer+.
       # [<tt>:index</tt>]
       #   Add an appropriate index. Defaults to false.
+      #   See #add_index for options.
```

Comment:
See `#add_index` for usage of this option.

Also lets move it on the line above.